response_operations_ui/templates/collection_exercise/ce-ci-section.html

```diff
@@ -30,6 +30,7 @@
   {% endif %}
   
     {% for ci_type in context["ci_table"]["ci_details"] %}
+      {% set ciLink = '<a href="' + ci_type.url + '" id="view-choose-upload-ci-' + ci_type.type + '">' + ci_type.link_text + '</a>' if ci_type.type != "cir" %}
```

Comment:
At first glance `ci_type.type != "cir"` was a bit confusing but it makes sense in the context of the page.